src/components/map/mapUtils.js

```diff
@@ -31,18 +31,15 @@ const buildTextStyle = function(title, highlight) {
     return text
 }
 
-export const buildPolygonStyle = function(title, highlight) {
-    const opacityFactor = highlight ? 1.5 : 1
```

Comment:
This was used for hovering effect, why removing it?

Reply:
In fact, I have changed the rendering of biodivsport data to be consistent with RcP:

* no hovering effect (and no text)
* yellow area instead of a blue one to match recommended protection zones.

Maybe I should post a screenshot of the rendering?

Reply:
I've seen it, don't worry.

If you change the ergo of these areas, I recommend a small message on forum

Reply:
I did the ergo the first time, nobody commented on it. I assume this change won't bother anyone 🙂 
For consistency, I cannot imagin another possibility right now.

Reply:
ok, let's go!